framework/wazuh/core/tests/data/configuration/default/journald1.conf

```diff
@@ -0,0 +1,6 @@
+<localfile>
+  <location>journald</location>
+  <log_format>journald</log_format>
+  <filter field="_SYSTEMD_UNIT">^cron.service$</filter>
+  <filter field="PRIORITY" ignore_if_missing="yes">[0-3]</filter>
+</localfile>
```

Comment:
```suggestion
</localfile>

```

Reply:
Added in ba463c2b3f963a736c4a8f466f6694363fe12bc6.